service/history/replicatorQueueProcessor.go

```diff
@@ -323,6 +323,70 @@ func GenerateReplicationTask(
 	return ret, newRunID, nil
 }
 
+// GenerateHistoryV2ReplicationTask generate replication task with history v2 replication tasks
```

Comment:
(In the long term) We should move these into a separate replication package?

Reply:
Yes. I think we want to keep the replication config for Kafka or RPC in the long term. So I guess a refactor is needed for this file before I move it to replication package